Makefile

```diff
@@ -78,7 +88,7 @@ run: welcome githooks-commit install build
 	@$(NODE) build/bundle-$(CALYPSO_ENV).js
 
 node-version:
-	@NPM_GLOBAL_ROOT=$(shell $(NPM) -g root) $(BIN)/check-node-version
+	@$(BIN)/check-node-version
```

Comment:
>thankfully there is no reason for using it. The node version is available through `process.version`

`semver` is used by [`bin/check-node-version`](https://github.com/Automattic/wp-calypso/blob/master/bin/check-node-version) for checking that the current Node version satisfies what we've defined as the minimum requirement as `engines.node` in `package.json`, prompting the developer to upgrade if they're using an earlier version.

>semver should not be available by default on any OS (it's an external package)

Which is true, but as a workaround to needing to `npm install semver` before checking the current version, we assume that the global `npm` package [defines it as a dependency](https://github.com/npm/npm/blob/ac9c70d5995a04614a68f554c0e88b5a48ce2858/package.json#L92) and call `semver` from its `node_modules` directory.

If this is enough of a problem, we could probably change this to:

```make
node-version:
	@$(NPM) install semver && $(BIN)/check-node-version
```

... and remove the shenanigans around `SEMVER_GLOBAL_PATH`